libs/lwt_platform/dune

```diff
@@ -0,0 +1,26 @@
+;; What is this? And why are we doing it?
+;; Lwt uses unix threads to run its stuff on, which means it doesn't work in Js_of_ocaml
+;; This library uses Dune's virtual modules. The way this works is that code can use
+;; this library's interface, lwt_platform, and get type checking and everything without
+;; having to choose a specific implementation. Then, when you compile, we can choose
```

Comment:
it's more when you link than compile, that is in the (executable section of a dune file

Reply:
Fair